Source/RageCommunity.Library/Graphics/Marker.cs

```diff
@@ -0,0 +1,114 @@
+﻿using Rage;
+using RageCommunity.Library.Wrappers;
+using System.Drawing;
+
+namespace RageCommunity.Library.Graphics
+{
+    public class Marker : ISpatial
```

Comment:
Missing description/summary. 
For example a developer who does not know what a GTA V Marker is. Is it a map icon?